src/pyproject_hooks/_in_process/_in_process.py

```diff
@@ -106,6 +106,13 @@ def find_spec(self, fullname, _path, _target=None):
 
         return spec
 
+    def find_distributions(self, context=None):
+        # Delayed import: Python 3.7 does not contain importlib.metadata
+        from importlib.metadata import DistributionFinder, MetadataPathFinder
+
+        context = DistributionFinder.Context(path=self.backend_path)
```

Comment:
Should this line be in an `if context is None` block so that passing a context in works correctly? Or perhaps it should intersect the search path from a passed in context with its own `self.backend_path`?

Reply:
I am not sure if the passed context will ever contain the `backend_path`, given that the importlib.metadats does not know about it[^1].

My intuition is that we can simply discard the given context, and other finders will handle it.

[^1]: at least for the current state of the implementation, in which `backend_path` is not added to `sys.path`.

Reply:
Ah, I see, it defaults to `sys.path` if no other path is passed. And neither of the places you can pass something else - `Distribution.discover()` or `distributions()` - are [mentioned in the docs](https://docs.python.org/3/library/importlib.metadata.html). So I think you're probably right that we can get away with ignoring the context argument.

Reply:
Agreed, this looks right to me.

Reply:
@jaraco is there any other optional part that needs to be implemented from `importlib.metadata`, `importlib.resources` point of view?

Reply:
This is all that's needed for `metadata`. I'm unsure if resources need an explicit hook. I'm 90% sure that because the `__spec__` is a `PathFinder`, it will have resources support.